### PR TITLE
Added getting metadata from a note id

### PR DESCRIPTION
**Pull request by darkdecoy**
Added getting metadata from a note id
Updated help menu and example for info command
Add json format option to info command

### CODE REVIEW

**Comment by ErikMichelson** [comment]
Hey @darkdecoy,
thanks for your contribution!

In order to get this merged, we need you to sign-off your commits according to the contribution guidelines for the HedgeDoc organization. You can find more information on this here: https://github.com/hedgedoc/hedgedoc/blob/develop/CONTRIBUTING.md#certificate-of-origin

**Comment by darkdecoy** [comment]
Added a sign off to each commit.

File: bin/hedgedoc [hunk 1]
```diff
@@ -70,6 +70,15 @@ Commands:
...
         View the current logged in user's list of accessed notes.
 
+    info <note_id> [--json]
+        View the metadata for a given note on the server.
+
+    info <note_id> [--json]
+        View the metadata for a given note on the server.
+
+    info <note_id> [--json]
+        View the metadata for a given note on the server.
```
Comment (ErikMichelson) [inline bin/hedgedoc:80]: This seems to be duplicated

File: bin/hedgedoc [hunk 2]
```diff
@@ -177,6 +188,75 @@ function import_note() {
...
+    local json="$2"
+
+    if [[ ! "$note_id" ]]; then
+        echo "Error: You must specify an export type and note id to export." >&2
+        echo "" >&2
+        echo "Usage: $SCRIPTNAME info <note_id> [--json]" >&2
+        echo "For usage exmaples, see: $SCRIPTNAME help" >&2
+        return 2
+    fi
+<<<<<<< HEAD
```
Comment (ErikMichelson) [inline bin/hedgedoc:202]: Here are some merge conflict markers, that shouldn't be committed

**Review by ErikMichelson — CHANGES_REQUESTED** [review]
The files from the export folder should not be committed as well

**Comment by darkdecoy** [comment]
made the requested changed

**Pull request closed without merging**